pkg/daemon/update.go

```diff
@@ -432,6 +432,26 @@ func (dn *Daemon) updateKernelArguments(oldConfig, newConfig *mcfgv1.MachineConf
 	return exec.Command("rpm-ostree", args...).Run()
 }
 
+// updateFIPS handles changes in FIPS
+func (dn *Daemon) updateFIPS(oldConfig, newConfig *mcfgv1.MachineConfig) error {
+	if oldConfig.Spec.FIPS != newConfig.Spec.FIPS {
```

Comment:
wait, shouldn't this be `==`?

Reply:
I know this is wip, just leaving also a comment that we still miss a call to `updateFIPS` 💪